src/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Grid, Button, Typography } from '@mui/material';
+import PageTitle from '../../layouts/PageTitle/PageTitle';
+import PageBlock from '../../layouts/PageBlock';
+import { useState } from 'react';
+
+const snark = ['Add your name!', "Shouldn't you do it yourself?", 'Seriously', 'go', 'do', 'it'];
+
+const CreditsPage: React.FC = () => {
+  const names = ['Kevin Chen']; // This is a list of names, add yours here!
```

Comment:
also can we make names be an array of `{ name: string; color: string; }`  and have that be the type annotation. This way they'll learn a bit about json and the page might be a little cooler if we render their name in that color (make a comment saying it has to be a hex)

Reply:
Done